### PR TITLE
GET_CONFIG: add raw/variable-processed contents of all config files

### PR DESCRIPTION
**Pull request by stapelberg**
We do this by adding to included_files as i3 processes the configs.

This should allow for easy debugging, without having to change how i3 processes
config files.

related to #4192

### CODE REVIEW

File: i3-msg/main.c [hunk 1]
```diff
@@ -262,32 +266,39 @@ int main(int argc, char *argv[]) {
...
-        yajl_status state = yajl_parse(handle, (const unsigned char *)reply, reply_length);
-        yajl_free(handle);
-
-        switch (state) {
-            case yajl_status_ok:
-                break;
-            case yajl_status_client_canceled:
-            case yajl_status_error:
-                errx(EXIT_FAILURE, "IPC: Could not parse JSON reply.");
+        if (raw_reply) {
```
Comment (Airblader) [inline i3-msg/main.c]: Is there something missing here? This case is empty.

Comment (stapelberg) [reply i3-msg/main.c]: Nothing missing, line 284 is what will print the reply :)

Comment (Airblader) [reply i3-msg/main.c]: In that case, why not just `if (!raw_reply)`?

Comment (stapelberg) [reply i3-msg/main.c]: Done

**Review by Airblader — APPROVED** [review]
LGTM, however the entry in the release notes is missing.

**Comment by stapelberg** [comment]
Done (plus the missing docs/ipc update, too).

**Pull request merged**